coc/abc.py

```diff
@@ -307,14 +349,17 @@ def _load_json(self, object_ids, english_aliases, lab_to_townhall):
                 continue
 
             # SC game files have "DisableProduction" true for all pet objects, which we want
-            if "DisableProduction" in meta and "pets" not in str(self.FILE_PATH):
+            if "DisableProduction" in meta and "pets" not in str(
+                    self.FILE_PATH):
                 continue
 
             # Little bit of a hacky way to create a "copy" of a new Troop object that hasn't been initiated yet.
-            new_item = type(self.data_object.__name__, self.data_object.__bases__, dict(self.data_object.__dict__))
+            new_item = type(self.data_object.__name__,
+                            self.data_object.__bases__,
+                            dict(self.data_object.__dict__))
             new_item._load_json_meta(
                 meta,
-                id=object_ids.get(supercell_name, 0),
+                id=object_ids.get(supercell_name, c+800),
```

Comment:
Magic integers should be constants with a comment.

Reply:
Okay, let's do that then?

Reply:
done

Reply:
okay it failed without id, so I just set the magic number to 0 like it used to be